packages/eas-json/src/EasJsonSchema.ts

```diff
@@ -72,10 +74,36 @@ const EasJsonBuildProfileSchema = CommonBuildProfileSchema.concat(
   })
 );
 
+const AndroidSubmitProfileSchema = Joi.object({
+  serviceAccountKeyPath: Joi.string(),
+  track: Joi.string().valid(...Object.values(ReleaseTrack)),
+  releaseStatus: Joi.string().valid(...Object.values(ReleaseStatus)),
+  verbose: Joi.boolean(),
+});
+
+const IosSubmitProfileSchema = Joi.object({
+  appleId: Joi.string(),
+  ascAppId: Joi.string(),
+
+  appleTeamId: Joi.string(),
+  itcTeamId: Joi.string(),
```

Comment:
The `itcTeamId` is not used anywhere. If you took it from here https://github.com/expo/eas-cli/blob/5c7862a685e550bffd5a6fa1b914774fc8143d59/packages/eas-cli/src/submissions/ios/AppProduce.ts#L21
it is a leftover and can be deleted. It is now automatically resolved based on `appleTeamId` by 🥓 Lane